source/uwp/SharedVisualizer/CustomElements/CustomInput.cs

```diff
@@ -5,7 +5,14 @@
 using System.Linq;
 using Windows.Data.Json;
 
-#if !USE_WINUI3
+#if USE_WINUI3
+using AdaptiveCards.Rendering.Winui3;
+using AdaptiveCards.ObjectModel.Winui3;
+using Microsoft.UI;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Media;
+#else
```

Comment:
Maybe. We can't do the thing where we alias "Microsoft.UI.Xaml" and "Windows.UI.Xaml" to just "xaml" ([source](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-directive#using-alias)). We could use `global` usings and have every namespace mentioned available to every file. Usually that's not great programming practice but in this instance it might be few enough namespaces and a small enough project. @asklar, do you have any thoughts?